### ISSUE TITLE
How can we help?

### ISSUE THREAD

**Issue by Pierstoval** [djhi/svelte-admin#2]
Hey :) This tool seems in its birth, and I'd like to know how we can help, because I'm interested :)

**Comment by djhi** [comment]
Hi Alex!

Indeed! I actually started to work on this about a year ago but had too much to do on react-admin.

I'm not trying to build a lib, at least not yet. I just wanted to learn about svelte and its differences with react. Building something similar to react-admin allows to learn svelte patterns on a domain I know quite well and explore things I might even port to react-admin later.

I'd be delighted to receive some help though but I don't have much time to put into this. Expect long delays for reviews, etc.

**Comment by Pierstoval** [comment]
Will let you take time to investigate on what you need then :) 

But if there's anything you'd like to test, feel free to call!

**Comment by Pierstoval** [comment]
Just for the sake of sharing it, I started working on a small admin system on this project: https://github.com/Orbitale/Compotes/tree/rewrite

It's a Tauri (Rust) + Svelte (Typescript) project, but the main goal is to have an admin-like interface in a desktop app, and the admin system is implemented in some Svelte routes and components, you can check it out.

I think I worked pretty well on the PaginatedTable now, but could not yet work on the add/edit part other than by doing everything "by hand" for now. But it's also a WIP 😄

**Comment by Pierstoval** [comment]
Hello again :wave: 

Thanks to a bit more time in research, and kind of a sponsoring from a client of mine, I managed to work on a more robust concept of a Svelte-admin package.

The code is there: https://github.com/Orbitale/SvelteAdmin

I started working on a Readme with a sample of what it looks, and the whole package was first built as a Typescript library with Svelte used as renderer.

I intend to make logic and rendering as separate as I can, so that it could also work with Web components for instance, or even plain DOM elements, as long as we can find a way to bind it. But so far, I focus on Svelte integration.

I also try to make sure the Svelte components can be built as web-components (so far, it works, but the output is very heavy).

And finally, I chose to take Carbon as main design system to avoid extra UI work, and it's really straightforward to use, especially since it already has [Svelte components](https://carbon-components-svelte.onrender.com/) (maintained by some IBM team members, as well as general community contributors).

If you or some teammates of yours are interested, I'm really open to discussion to make this a usable approach :+1:
